Libraries/src/Amazon.Lambda.APIGatewayEvents/APIGatewayProxyRequest.cs

```diff
@@ -332,6 +337,69 @@ public class RequestIdentity
             /// The user
             /// </summary>
             public string User { get; set; }
+
+
+            /// <summary>
+            /// Properties for a client certificate.
+            /// </summary>
+            public ProxyRequestClientCert ClientCert { get; set; }
+        }
+
+        /// <summary>
+        /// Container for the properties of the client certificate.
+        /// </summary>
+        public class ProxyRequestClientCert
```

Comment:
What is the reason the ProxyRequestClientCert, ClientCertValidity, and probably other classes are duplicated?

Reply:
They are duplicate because I can't control how API Gateway models the events in the future. Currently they are the same but in the future they might differ between the REST API event format and the HTTP API event format.